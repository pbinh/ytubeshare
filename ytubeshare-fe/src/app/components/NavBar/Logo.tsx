import { Col, Row } from 'antd';
import * as React from 'react';
import styled from 'styled-components/macro';
import './style.scss'

export function Logo() {
  return (
    <Row style={{height: '100%', paddingLeft: 10}} align={'middle'}>
      <Col>
        <Title>Remitano</Title>
      </Col>
      <Col xs={0}>
        <Description>Youtube Sharing Website</Description>
      </Col>
    </Row>
  );
}

const Wrapper = styled.div`
  display: flex;
  align-items: center;
`;

const Title = styled.div`
  font-size: 1.25rem;
  color: ${p => p.theme.text};
  font-weight: bold;
`;

const Description = styled.div`
  font-size: 0.875rem;
  color: ${p => p.theme.textSecondary};
  font-weight: normal;
`;
