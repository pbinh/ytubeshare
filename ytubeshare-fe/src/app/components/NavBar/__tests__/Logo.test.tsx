import * as React from 'react';
import { render } from '@testing-library/react';
import { Logo } from '../Logo';
import { createRenderer } from 'react-test-renderer/shallow';

const shallowRenderer = createRenderer();

describe('<Logo />', () => {
  it('should match snapshot', () => {
    shallowRenderer.render(
      <Logo />
    );
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
