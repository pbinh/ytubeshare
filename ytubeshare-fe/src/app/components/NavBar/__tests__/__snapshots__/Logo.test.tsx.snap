// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should match snapshot 1`] = `
<Row
  align="middle"
  style={
    Object {
      "height": "100%",
      "paddingLeft": 10,
    }
  }
>
  <Col>
    <Logo__Title>
      Remitano
    </Logo__Title>
  </Col>
  <Col
    xs={0}
  >
    <Logo__Description>
      Youtube Sharing Website
    </Logo__Description>
  </Col>
</Row>
`;
